libs/sdk-ui-dashboard/api/sdk-ui-dashboard.api.md

```diff
@@ -1606,7 +1607,7 @@ export interface DashboardInsightWidgetExportResolved extends IDashboardEvent {
 
 // @beta
 export interface DashboardInsightWidgetExportResolvedPayload {
-    resultUri: string;
+    result: IExportResult;
```

Comment:
describe this property

Reply:
I updated the comment as it was wrong but strange that it is not here in the exported interface doc as the prop was  already documented. See `libs/sdk-ui-dashboard/src/model/events/insight.ts:64`